src/main.rs

```diff
@@ -0,0 +1,19 @@
+mod api_client;
+mod error;
+mod serde;
+mod types;
```

Comment:
I think the CI is getting confused by these?

either way you should remove them

Reply:
ok